editor/src/components/text-utils.ts

```diff
@@ -247,7 +247,7 @@ export async function measureTextFieldNew(element: ElementInstanceMetadata): Pro
       },
       scale: 1,
     }
-    const textElement = <Text {...textFieldProps} />
+    const textElement = React.createElement(Text, textFieldProps)
```

Comment:
This one is definitely cheating :eyes:

Reply:
This whole function put the fear into me, promise based modifying of the DOM that then triggers actions to be dispatched back into the editor.

Reply:
Ha ok I just followed this through and see why you've done this! I was going to ask for this to be removed but given how this is used I'd say it's fine to leave as is.